src/frontend/web_application/src/components/MessageList/components/Message/style.scss

```diff
@@ -1,49 +1,46 @@
 @import '../../../../styles/vendor/bootstrap_foundation-sites';
+@import '../../../../styles/object/o-avatar';
 
 .m-message {
   @include flex-grid-row($size: expand);
-  margin-bottom: $co-grid__gutter;
-  font-size: .9rem;
+  font-size: $co-font__size--small;
+
+  @include breakpoint(small) { margin-bottom: $co-grid__gutter--small; }
+  @include breakpoint(medium) { margin-bottom: $co-grid__gutter; }
+
+  &__avatar-col {
+    @include breakpoint(small) {
```

Comment:
I have no special guideline about how to organize the code, generally I like to have normal representation first then each breakpoints  not mixed e.g


```
.m-block {
  // standard positioning (for small devices)
  // ...
  &__elem {
    // ...
  }

  // breakpoints
  @include breakpoint(medium) {
    & { // ... }
    &__elem { // ... }
  }
  
  @include breakpoint(large) {
    &__elem { // ... }
  }
}
```

It's up to you.